### PR TITLE
Fix pylint issue

### PR DESCRIPTION
**Pull request by harshbafna**
Address #118 

This PR consists of following

- Changes for errors reported by pylint.
- Updated torchserve_sanity script to use latest version of pylint and execute pylint command
- Unused code cleanup
- UT fixes

### CODE REVIEW

File: ts/model_loader.py [hunk 1]
```diff
@@ -23,14 +24,8 @@ class ModelLoaderFactory(object):
     """
 
     @staticmethod
-    def get_model_loader(model_dir):
-        manifest_file = os.path.join(model_dir, "MAR-INF/MANIFEST.json")
-        if os.path.exists(manifest_file):
-            return TsModelLoader()
-        elif os.path.exists(os.path.join(model_dir, "MANIFEST.json")):
-            return LegacyModelLoader()
```
Comment (mycpuorg) [inline ts/model_loader.py:31]: Not sure if this is a lint rule we set up.
Not clear why this isn't necessary. Could you please explain?

**Comment by mycpuorg** [comment]
Also this branch is out of date

File: ts/model_loader.py (hunk 1 above)
Comment (harshbafna) [reply ts/model_loader.py:31]: The LegacyModelLoader was ported from MMS and is no more in use in TorchServe.

**Comment by mycpuorg** [comment]
The code changes here are recommended from pylint checks. I have tested with a basic run [here](https://github.com/pytorch/serve/pull/256#issuecomment-635011730) with these changes on `staging_0_1_1` . Adding `v0.1.1_tested` label based on this since there isn't specific case to be tested here.

**Pull request merged**